ordered_model/admin.py

```diff
@@ -248,14 +248,23 @@ def get_preserved_filters(cls, request):
         return ''
 
     def move_up_down_links(self, obj):
-        if obj.id:
-            order_obj_name = 'obj'
-            if obj._get_order_with_respect_to() is not None:
-                order_obj_name = obj._get_order_with_respect_to().id
+        if not obj.id:
+            return ''
+
+        # Find the fields which refer to the parent model of this inline, and
```

Comment:
Happy to merge this as is, but this bit of logic probably makes sense to be factored into its own function somewhere. It's hard to parse what is going on here.